drivers/net/ethernet/intel/idpf/Makefile

```diff
@@ -1,18 +1,29 @@
 # SPDX-License-Identifier: GPL-2.0-only
-# Copyright (C) 2023 Intel Corporation
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Copyright year is year of coding, not year of modification.

Reply:
Pavan had a comment on this to be changed. @plinga1 Can you please confirm on this.

Reply:
There was a comment when we upstreamed IDPF to update the copyright year to the current year the driver makes into the upstream. Couldnt find the comment but you may consult Tony for additional info.

Reply:
See here is my past experience with Copyright. When the file is first coded, the copyright line shows that year e.g.
Copyright (C) 2021 Rafał Miłecki <rafal@milecki.pl>

Now suppose above file is modified again in 2022,  the copyright should be 
Copyright (C) 2021-2022 Rafał Miłecki <rafal@milecki.pl>

Then the file gets modified in 2024 (means not modified in 2023), the copyright should read:
Copyright (C) 2021-2022, 2024 Rafał Miłecki <rafal@milecki.pl>

Essentially, every year that you modify the file should be listed via range (using dash) or explicitly

Reply:
I initially talked to Tony about it, and I had left it to the original code date of this driver based on the feedback from him. I will ask him again about it.

Reply:
I have caused a problem here.. some of the files I moved the year based on Pavan's comments so some of them would end up showing as 2024

Reply:
No worry, I will fix them again.